roboteam_ai/src/skills/Keeper.cpp

```diff
@@ -112,6 +112,17 @@ Vector2 Keeper::computeBlockPoint(Vector2 defendPos) {
         blockPos = Vector2(goalPos.x + constants::KEEPER_POST_MARGIN, goalwidth/2
                 *signum(defendPos.y)); // Go stand at one of the poles depending on the side the defendPos is on.
     }
+    //Interface visualization:
+    std::vector<std::pair<rtt::Vector2, QColor>> displayColorData;
+    std::pair<rtt::Vector2, QColor> A=std::make_pair(blockPos,Qt::red);
+    displayColorData.push_back(A);
+    displayColorData.emplace_back(std::make_pair(blockLineStart,Qt::red));
```

Comment:
dit kan ook korter met `insert`

Reply:
I think it is too much information in one line.